lib/charms/traefik_k8s/v2/ingress.py

```diff
@@ -132,6 +144,7 @@ def _on_ingress_revoked(self, event: IngressPerAppRevokedEvent):
         "name": str,
         "strip-prefix": bool,
         "redirect-https": bool,
+        "scheme": Scheme,
```

Comment:
The enum doesn't seem to work too well as it leaks everywhere.
I wonder if we should just stick with `str` for now until we go all-in on pydantic.
If traefik crashes for invalid scheme, then we may be better off to check it later on in the charm?